.github/workflows/issue-pr-trigger.yml

```
@@ -0,0 +1,49 @@
+jobs:
+  notify-lark:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
+      - uses: actions/setup-node@cdca7365b2dadb8aad0a33bc7601856ffabcc48e # v4
+        with:
+          node-version: "20.19.0"
+      - name: Check if triggered by project member
+        id: check_member
+        if: github.event.sender.type == 'User' && github.actor != 'renovate'
+        run: |
```

actionlint (1.7.4)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)